target/linux/ipq806x/patches-5.4/100-ipq8064-dtsi-tweaks.patch

```diff
@@ -0,0 +1,206 @@
+--- a/arch/arm/boot/dts/qcom-ipq8064.dtsi	2020-07-03 10:44:22.410037215 +0200
++++ b/arch/arm/boot/dts/qcom-ipq8064.dtsi	2020-07-03 11:04:39.101098627 +0200
+@@ -8,6 +8,8 @@
+ #include <dt-bindings/reset/qcom,gcc-ipq806x.h>
+ #include <dt-bindings/soc/qcom,gsbi.h>
+ #include <dt-bindings/interrupt-controller/arm-gic.h>
++#include <dt-bindings/mfd/qcom-rpm.h>
++#include <dt-bindings/clock/qcom,rpmcc.h>
```

Comment:
I'm not sure where these two lines are actually needed, I suspect they can be moved to 1.0.dtsi at least ...